app/components/recommended-video.js

```diff
@@ -0,0 +1,109 @@
+import {inject as service} from '@ember/service';
+import Component from '@ember/component';
+import {run} from '@ember/runloop';
+import NoScrollMixin from '../mixins/no-scroll';
+import jwPlayerAssets from '../modules/jwplayer-assets';
+import {track} from '../utils/track';
+import extend from '../utils/extend';
+
+export default Component.extend(NoScrollMixin, {
+	logger: service(),
+
+	classNames: ['recommended-video'],
+	classNameBindings: ['isExtended', 'isReady', 'isClosed'],
+
+	playlistItem: null,
+	playlistItems: null,
+
+	init() {
+		this._super(...arguments);
+
+		run.later(() => {
+			this.initRecommendedVideo();
+		}, 5000);
```

Comment:
did we get this number from designer/product or it is just example one?

Reply:
it's just an example value.